README.md

```diff
@@ -1,3 +1,159 @@
 # Holochain HTTP Gateway
 
 The Holochain HTTP Gateway for providing a way to bridge from the web2 world into Holochain
+
+## Running HTTP Gateway locally
+
+### Prerequisites
+
+Enter the Nix `devShell` with `nix develop` or make sure that you have `rustup`
+and `holochain-cli` installed.
+
+### Instructions
+
+Clean the Holochain conductor sandbox with:
+
+```bash
+hc sandbox clean
+```
+
+Create a new sandboxed conductor with the lair keystore running in the
+Holochain process, entering a passphrase when prompted:
+
+```bash
+hc sandbox create --in-process-lair
```

Comment:
By default it's the in process lair, so you can drop the option.

Reply:
When I run this without the flag I get a conductor config that contains

```yaml
keystore:
  type: lair_server
  connection_url: unix:///tmp/nix-shell.m0A5YR/OZ8Ge1DWCZIqjmBvgyw4m/ks/socket?k=Ec9bTgTawgroTdlePi5SHP9uYJfoWlmNFK9YWIeq53E
  ```

I agree it should be the default, and maybe is from 0.5?

Reply:
Ah, okay! Yes, should be.